modules/dailymotionBidAdapter.js

```diff
@@ -72,4 +75,28 @@ export const spec = {
   interpretResponse: serverResponse => serverResponse?.body ? [serverResponse.body] : [],
 };
 
+function getVideoMetadata(bidRequest) {
+  const videoAdUnit = deepAccess(bidRequest, 'mediaTypes.video', {});
+  const videoBidderParams = deepAccess(bidRequest, 'params.video', {});
+
+  const videoParams = {
+    ...videoAdUnit,
+    ...videoBidderParams // Bidder Specific overrides
+  };
+
+  const videoMetadata = {
+    description: videoParams.description ? videoParams.description : '',
+    duration: videoParams.duration ? videoParams.duration : 0,
+    iabcat2: videoParams.iabcat2 ? videoParams.iabcat2 : '',
+    id: videoParams.id ? videoParams.id : '',
+    lang: videoParams.lang ? videoParams.lang : '',
+    private: videoParams.private ? videoParams.private : false,
+    tags: videoParams.tags ? videoParams.tags : '',
+    title: videoParams.title ? videoParams.title : '',
+    topics: videoParams.topics ? videoParams.topics : '',
```

Comment:
To follow up on previous comment, you can remove ternary operator here and use a logical OR instead
`topics: videoParams.topics || '',`

Reply:
I included this in my changes to move the xid